subworkflows/nf-scil/anatomical_segmentation/tests/main.nf.test

```diff
@@ -35,7 +35,8 @@ nextflow_workflow {
                 input[0] = LOAD_DATA.out.test_data_directory.map{
                     test_data_directory -> [
                         [ id:'test', single_end:false ], // meta map
-                        file("\${test_data_directory}/anat/anat_image.nii.gz", checkIfExists: true)
+                        file("\${test_data_directory}/anat/anat_image.nii.gz", checkIfExists: true),
+                        []
```

Comment:
Here, you added an empty input to the `nf-test` file, but I don't see that you added a `lesion` optional input in the `main.nf` of the actual subworkflow file.

Reply:
It's because the subworflow take the same inputs as `freesurferseg` and `fastseg`, and when I test the modules it also test the subworflow. So the lesion option is directly implemented in modules.

Reply:
Yepp, you are right. You can discard my comment!